### DISCUSSION TITLE
Unable to lock files

### DISCUSSION THREAD

**Discussion by argo-1** [ProjectBorealis/UEGitPlugin#132] [Q&A] [answered]
Hello. I have a UE 5.1 project on bitbucket, cloned with https with this plugin added as a submodule. 

When my teammates and I connect to source control and modify a file, it says checked out locally with a red tick but this status never seems to reach the other users. How can we solve this?

**Comment by mastercoms** [comment, marked as answer]
https://jira.atlassian.com/browse/BCLOUD-14454

BitBucket does not support LFS locking

**Reply by mastercoms** [reply]
You can use GitHub, GitLab, Azure DevOps Repos, a Gitea instance

**Reply by argo-1** [reply]
https://confluence.atlassian.com/bitbucketserver/working-with-git-lfs-files-970595880.html

Bitbucket seems to indicate that they do indeed have git lfs locking now. Is this a problem from the plugin's side, or maybe it doesn't work with this plugin's pipeline?

**Reply by argo-1** [reply]
Ahhh, bitbucket server has it but not bitbucket cloud, I see.
